--- rapidjson-0.12~git20140926.orig/build/premake4.lua
+++ rapidjson-0.12~git20140926/build/premake4.lua
@@ -122,9 +122,9 @@ solution "test"
             "../include/",
             "../thirdparty/gtest/include/",
             "../thirdparty/",
-            "../thirdparty/jsoncpp/include/",
+            "/usr/include/jsoncpp",
             "../thirdparty/libjson/",
-            "../thirdparty/yajl/include/",
+            "/usr/include/yajl/",
         }
 
         libdirs "../thirdparty/lib"
