--- i/build/premake4.lua
+++ w/build/premake4.lua
@@ -72,13 +72,11 @@ solution "test"
         defines { "GTEST_HAS_PTHREAD=0" }
 
         files { 
-            "../thirdparty/gtest/src/gtest-all.cc",
-            "../thirdparty/gtest/src/**.h",
+            "/usr/src/gtest/src/gtest-all.cc",
         }
 
         includedirs {
-            "../thirdparty/gtest/",
-            "../thirdparty/gtest/include",
+            "/usr/src/gtest",
         }
 
         setTargetObjDir("../thirdparty/lib")
@@ -98,7 +96,6 @@ solution "test"
         
         includedirs {
             "../include/",
-            "../thirdparty/gtest/include/",
         }
 
         libdirs "../thirdparty/lib"
@@ -119,8 +116,6 @@ solution "test"
         
         includedirs {
             "../include/",
-            "../thirdparty/gtest/include/",
-            "../thirdparty/",
         }
 
         libdirs "../thirdparty/lib"
